src/utils/gauges.ts

```diff
@@ -37,7 +39,7 @@ export type GaugeReward = {
 }
 
 export type PoolAddressToGauge = {
-  [poolAddress: string]: Gauge | undefined
+  [poolAddress: string]: Gauge
```

Comment:
no need to change this, the previous type is more acccurate because `gauges[anyAddress]` may be undefined.

Instead, you can override the type for Object.values in `react-app-env.d.ts` with

```
interface ObjectConstructor {
  keys<T>(o: T): ObjectKeys<T>
  values<T>(o: { [s: string]: T } | ArrayLike<T>): NonNullable<T>[]
  // eslint-disable-next-line @typescript-eslint/ban-types
  values(o: {}): any[]
}
```

Reply:
I addressed in a PR here https://github.com/saddle-finance/saddle-frontend/pull/1049